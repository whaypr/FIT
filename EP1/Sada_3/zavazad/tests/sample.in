6

5 1 1 1 1 1
2 1 2
3 0 1 1

9 0 1 2 0 1 2 0 1 2
6 0 0 0 1 1 2
6 0 0 0 1 1 0