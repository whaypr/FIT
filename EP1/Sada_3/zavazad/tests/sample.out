Je treba 0 presunu.
Je treba 1 presunu.
Je treba 3 presunu.
Je treba 170 presunu.
Je treba 7 presunu.
Je treba 6 presunu.
